python_modules/dagster/dagster/core/asset_defs/decorators.py

```diff
@@ -67,7 +67,7 @@ def my_asset(my_upstream_asset: int) -> int:
     if callable(name):
         return _Asset()(name)
 
-    def inner(fn: Callable[..., Any]) -> OpDefinition:
+    def inner(fn: Callable[..., Any]) -> MultiAssetDefinition:
```

Comment:
Very much a nitpick, but I find it kinda unaesthetic that `@asset` and `@multi_asset` both return a `MultiAssetDefinition`, because the existence of a `MultiAssetDefinition` implies the existence of an `AssetDefinition`. I get that having just a single asset is just a special case of having multiple assets (and so it's even worse to have some AssetDefinition that subclasses MultiAssetDefinition), but could we instead call `MultiAssetDefinition` something like `AssetsDefinition`?

It feels a little more natural to me that `AssetsDefinition` might define one or more assets than `MultiAssetDefinition` having that behavior (which seems more like it's guaranteed to define more than one).

Reply:
Hmm yeah, I toyed with the name `AssetsDefinition` a little bit. One concern is that if we do choose to introduce `AssetDefinition`, distinguishing between the two might be annoying.  We can cross the bridge when we come to it though - I'll make the change you're suggesting.